api.swagger.yaml

```diff
@@ -16750,9 +16773,10 @@ components:
           example: '01234567-89ab-cdef-1234-56789abcdef0'
         author_uuid:
           description: 'Invotra UUID of the author.'
-          example: '01234567-89ab-cdef-1234-56789abcdef0'
           type: string
           format: uuid
+          example: '01234567-89ab-cdef-1234-56789abcdef0'
+          deprecated: true
```

Comment:
Happy with this on the read requests. Could we keep author_uuid in on the POST request?